Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTestBase.m

```diff
@@ -25,63 +25,83 @@
 //
 
 #import "BUYClientTestBase.h"
-#import "BUYTestConstants.h"
 #import "NSArray+BUYAdditions.h"
 
+NSString * const kBUYTestEmail = @"customer_email";
+NSString * const kBUYTestPassword = @"customer_password";
+NSString * const kBUYTestOrderIds = @"customer_order_ids";
+NSString * const kBUYTestGiftCardCode11 = @"gift_card_code_11";
+NSString * const kBUYTestGiftCardCode25 = @"gift_card_code_25";
+NSString * const kBUYTestGiftCardCode50 = @"gift_card_code_50";
+NSString * const kBUYTestInvalidGiftCardCode = @"invalid_gift_card_code";
+NSString * const kBUYTestExpiredGiftCardCode = @"expired_gift_card_code";
+NSString * const kBUYTestExpiredGiftCardID = @"expired_gift_card_id";
+NSString * const kBUYTestDiscountCodeValid = @"discount_valid";
+NSString * const kBUYTestDiscountCodeExpired = @"discount_expired";
+NSString * const kBUYTestProductIdsCommaSeparated = @"product_ids_comma_separated";
+NSString * const kBUYTestCollectionIds = @"collection_ids";
```

Comment:
Can we make this `static NSString * const ...` instead to match the definitions for `BUYDomainName` and others? The `extern` declarations are in `BUYTestConstants.h` but the file doesn't seem to be imported anywhere, or is it?